experimental/yarpl/include/yarpl/Flowables.h

```diff
@@ -104,6 +104,18 @@ class Flowables {
     return Flowable<T>::create(std::move(lambda));
   }
 
+  template <typename T>
+  static Reference<Flowable<T>> repeat(T value) {
+    auto lambda = [v = std::move(value)](
+        Subscriber<T> & subscriber, int64_t requested) mutable {
```

Comment:
Can we give a negative value into this function? Should we have any check for negative?